docs/import/manage_disaster_flexible.js

```diff
@@ -380,63 +524,74 @@ async function initializeFlexibleDisaster(assetData) {
         break;
     }
   }
-  startPending();
-  try {
-    await Promise.all([
-      initializeGeography(),
-      initializePoverty(),
-      initializeBuildings(),
-    ]);
-  } finally {
-    finishPending();
+  initializeChecker.maybeStartPending();
+  const [result] = await Promise.all([
+    initializeGeography(),
+    initializePoverty(),
+    initializeBuildings(),
+  ]);
+  if (result) {
+    // Just check one value to see if it succeeded: others must be the same.
+    initializeChecker.finishPending();
```

Comment:
Is the finishPending what's checking the value?

Reply:
No, moved line up to hopefully make clearer.

Reply:
maybe rename to geographyResult